pandas/tests/arrays/test_integer.py

```diff
@@ -587,6 +587,14 @@ def test_cross_type_arithmetic():
     tm.assert_series_equal(result, expected)
 
 
+def test_arith_extension_array_values():
```

Comment:
Can you move the test to `pandas/tests/extension/base/ops.py`, as this is not a specific failure for IntegerArray, it is relevant for all extension types.